install.sh

```diff
@@ -188,15 +188,15 @@ check_deps() {
 }
 
 check_python() {
-  if which python2.7 &> /dev/null; then
+  if type -P python2.7 &> /dev/null; then
     PYTHONVERSION="2.7"
     PYTHON="python2.7"
-  else
-    PYTHONVERSION=`python --version 2>&1`
-    PYTHON=python
+  elif type -P python &> /dev/null; then
+    PYTHONVERSION=`python -c 'import sys; print(".".join(map(str, sys.version_info[:2])))'`
```

Comment:
Do you know which python versions support print function? I remember there were error reports about it.